webapp/components/Chart/utils/ChartType.ts

```diff
@@ -0,0 +1,4 @@
+export enum ChartType {
+  bar = 'bar',
+  pie = 'pie',
+}
```

Comment:
Should/could this be somehow written with `keyof ChartTypeRegistry`?
(Where `ChartTypeRegistry` is an object with the correct keys)

Although, we don't support other charts at the moment

Reply:
let's keep it simple like this for now